### PR TITLE
Feat/sidebar meta info

### PR DESCRIPTION
**Pull request by tomADC443**
This PR includes the changes from "Feat/label with meta info on hover" So review that first, merge it, and then review this. THX

### CODE REVIEW

File: src/components/atoms/AreaDetails/AreaDetails.tsx [hunk 1]
```diff
@@ -43,8 +50,16 @@ export const AreaDetails: React.FC<AreaDetailsProps> = ({ areaSize, firstAnalysi
...
+                        <td>{timestampsCount}</td>
                     </tr>
+                    {timestampWithSignificantPlastic && (
+                        <tr>
+                            <td>
+                                <p>Number of days without plastic found:</p>
+                            </td>
+                            <td>{timestampsCount - timestampWithSignificantPlastic}</td>
+                        </tr>
+                    )}
```
Comment (emely3h) [inline src/components/atoms/AreaDetails/AreaDetails.tsx:62]: Are we only showing this if timestampWithSignificantPlastic > 0? Wouldn't it be a useful information to know that there there was no day without any plastic detected?:)

File: src/components/organisms/MapBoxMap/MapboxMap.tsx [hunk 2]
```diff
@@ -63,6 +64,17 @@ const MapboxMap: React.FC = () => {
...
+    } = useQuery({
+        queryFn: async () => await fetchCurrentAoiMetaData(currentAoiId!),
+        enabled: Boolean(currentAoiId),
+        queryKey: ['currentAoiMetaData', currentAoiId],
+        refetchOnWindowFocus: false,
+    })
+
     // const openSidebar = () => {
     //     setIsSidebarOpen(!isSidebarOpen)
     // }
```
Comment (emely3h) [inline src/components/organisms/MapBoxMap/MapboxMap.tsx]: If we don't need this I would just remove it

File: src/services/mapService.tsx [hunk 3]
```diff
@@ -59,6 +59,26 @@ export async function fetchRegionDatetimes(aoiId: AoiId): Promise<number[]> {
...
+
+        const currentAoiMetaData: any = await JSON.parse(await response.json())
+        const metaData: CurrentAoiMetaData = {
+            timestampWithSignificantPlastic: currentAoiMetaData.features[0].properties.timestamp_with_plastic_count,
+        }
+        console.log('currentaoiMeatdata2 response', currentAoiMetaData)
+        console.log('currentAoiMetaData2', metaData)
+        return metaData
+    } catch (error) {
+        console.error('Error retrieving currentAOI meta data:', error)
```
Comment (emely3h) [inline src/services/mapService.tsx]: probably some debugging leftovers:)

File: src/services/regionLayerService.tsx [hunk 4]
```diff
@@ -69,6 +41,7 @@ function addUnclusteredPointHover(map: mapboxgl.Map) {
             }
             const startDateString = new Date(startDate * 1000).toLocaleString(undefined, dateOptions)
             const endDateString = new Date(endDate * 1000).toLocaleString(undefined, dateOptions) // Convert to human readable date, with browser timezone
+            console.log('Feature', event.features![0])
```
Comment (emely3h) [inline src/services/regionLayerService.tsx]: probably some debugging leftovers:)

**Review by emely3h — APPROVED** [review]

**Comment by emely3h** [comment]
Nice works:)

I played around a bit, I noticed that for aois over the river I did not find a timestamp where something was detected, even though I clicked around a bit and theoretically half of the timestamps should have something. Is it because our detection did not work properly or is it just coincidence that I selected only the onces where nothing was detected?

File: src/components/atoms/AreaDetails/AreaDetails.tsx (hunk 1 above)
Comment (tomADC443) [reply src/components/atoms/AreaDetails/AreaDetails.tsx:62]: we are always showing it, even when there are 0 days without plastic :) 
The conditional rendering is just to make sure that we only display this if we have the information about the `timestampWithSignificantPlastic` which we should have usually :)

File: src/components/organisms/MapBoxMap/MapboxMap.tsx (hunk 2 above)
Comment (tomADC443) [reply src/components/organisms/MapBoxMap/MapboxMap.tsx]: Yes! I deleted it

File: src/services/mapService.tsx (hunk 3 above)
Comment (tomADC443) [reply src/services/mapService.tsx]: ups.... thanks! I cleaned them up!

**Comment by tomADC443** [comment]
> Nice works:)
> 
> I played around a bit, I noticed that for aois over the river I did not find a timestamp where something was detected, even though I clicked around a bit and theoretically half of the timestamps should have something. Is it because our detection did not work properly or is it just coincidence that I selected only the onces where nothing was detected?

I think you were unlucky, i found a couple, will check that again when we process the new regions

**Pull request merged**